src/components/Footer.jsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames';
+import "./../styles/footer.css";
+
+const Footer = ({ activeStep, handleNext, handleBack }) => {
+
+    const nextButton = classNames({
+        'next-button': true,
+        'confirm-background': activeStep === 3
+    });
+
+  return (
+    <div className="footer-container">
+      {activeStep !== 0 && (
+        <button onClick={handleBack} className="back-button">
+          Go Back
+        </button>
+      )}
+      <button onClick={handleNext} className={nextButton}>
+        {activeStep === 3 ? "Confirm" : "Next Step"}
```

Comment:
try not to hardcode the steps.

you want to have activeStep.length  - 1 or something because the LAST step is the one with the confirm. Not just step 3. It just so happens step 3 is last in your case.